src/structures/APIMessage.js

```diff
@@ -88,6 +88,13 @@ class APIMessage {
       content = Util.resolveString(this.options.content);
     }
 
+    const disableMentions = typeof this.options.disableMentions === 'undefined' ?
+      this.target.client.options.disableMentions :
+      this.options.disableMentions;
+    if (disableMentions) {
+      content = (content || '').replace(/@/g, '@\u200b');
```

Comment:
Maybe you could use `Util.removeMentions` here?

Reply:
Good idea.